springfield/firefox/redirects.py

```diff
@@ -105,4 +105,6 @@ def mobile_app(request, *args, **kwargs):
         r"/(?P<channel>release|aurora)notes/(?P<page>[\/\w\.-]+)?$",
         "http://website-archive.mozilla.org/www.mozilla.org/firefox_releasenotes/en-US/{prod}/{vers}/{channel}notes/{page}",
     ),
+    # Bug 868182
+    redirect(r"^mobile/faq/?$", firefox_mobile_faq, query=False),
```

Comment:
`firefox_mobile_faq` seems actually quite bizarre to keep; but since it's been in the refracts, it might be a hint it actually was perhaps used at some point with fxc. Can't really find any source that would confirm these URLs were ever only on mozorg or also on fxdotcom … so they're probably to stay?

Reply:
I always pause a bit when I see `query=False` ~ it does something else than what it may seem: it means "add these query parameters: none", not "remove any query string when redirecting" (it is the default value IIUC), so it being often added on redirects that actually work with some query params is a little puzzling. ~ Q: does this serve any purpose I failed to infer, or would it be okay if, when I'm in the neighborhood (as I have a few more basic redirect tests that I ought to add) and removing the defaults doesn't break anything, if I removed these in cases they actually make understanding the rules harder?

Reply:
yes, I gotta try and test that assumption first anyway.